dedupe/ckan_api.py

```diff
@@ -111,26 +111,26 @@ def get_dataset(self, organization_name, identifier, is_collection, sort_order='
 
         return results[0]
 
-    def get_duplicate_identifiers(self, organization_name, is_collection):
+    def get_duplicate_guids(self, organization_name, is_collection):
```

Comment:
Add a new optional parameter called `identifier_type` that defaults to `identifier`, and then use that parameter in place of the first identifier on line 121.
Will also need to parameterize the return on line 128.